oracle/docker-compose-erc-native.yml

```diff
@@ -81,6 +82,20 @@ services:
     entrypoint: yarn worker:swap-tokens
     networks:
       - net_rabbit_bridge_swap_tokens_worker
+  bridge_convert_to_chai_worker:
+    cpus: 0.1
+    mem_limit: 500m
+    build:
+      context: ..
+      dockerfile: oracle/Dockerfile
```

Comment:
Please update this branch with the latest changes from `develop` to include the changes from https://github.com/poanetwork/tokenbridge/pull/280

The `build` tag should be replaced with the image from docker hub.
```yaml
image: poanetwork/tokenbridge-oracle:latest
```

Reply:
Done